tests/test_sdist.py

```diff
@@ -50,10 +50,12 @@ def test_ignore_based_on_marker(tmp_context: WorkContext):
     assert version == ""
 
 
-def test_invalid_wheel_file_exception(tmp_path: pathlib.Path):
+@patch("fromager.sources.download_url")
+def test_invalid_wheel_file_exception(mock_download_url, tmp_path: pathlib.Path):
+    mock_download_url.return_value = str(tmp_path / "test" / "fake_wheel.txt")
     fake_dir = tmp_path / "test"
     fake_dir.mkdir()
-    test_url = "https://github.com/python-wheel-build/fromager/blob/main/README.md"
-
+    text_file = fake_dir / "fake_wheel.txt"
+    text_file.write_text("This is a test file")
     with pytest.raises(zipfile.BadZipFile):
-        sdist._download_wheel_check(fake_dir, test_url)
+        sdist._download_wheel_check(fake_dir, mock_download_url)
```

Comment:
This part doesn't look quite right. `mock_download_url` is a mock of a function, but this argument should be a string URL, right?

Reply:
Hmm, I am unable to figure out what should go as parameter? Should I pass in the file name that I create? I am not sure